tests/end-to-end/external-solution-integration/pkg/testsuite/create_service_binding_usage.go

```diff
@@ -1,16 +1,17 @@
 package testsuite
 
 import (
-	serviceBindingUsageApi "github.com/kyma-project/kyma/components/service-binding-usage-controller/pkg/apis/servicecatalog/v1alpha1"
-	serviceBindingUsageClient "github.com/kyma-project/kyma/components/service-binding-usage-controller/pkg/client/clientset/versioned/typed/servicecatalog/v1alpha1"
-	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/helpers"
-	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/retry"
-	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/step"
 	"github.com/pkg/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
-	knativeEventingV1alpha1 "knative.dev/eventing/pkg/client/clientset/versioned/typed/eventing/v1alpha1"
-	knativeMessagingV1alpha1 "knative.dev/eventing/pkg/client/clientset/versioned/typed/messaging/v1alpha1"
+	eventingv1alpha1clientset "knative.dev/eventing/pkg/client/clientset/versioned/typed/eventing/v1alpha1"
```

Comment:
Please cluster knative.dev and k8s.io

Reply:
all imports are clustered using `goimports -local github.com/kyma-project,github.com/kyma-incubator`
this results in:
1 cluster of go standard library
1 cluster for other imports
1 cluster for local imports

that is all the tool can do, imho thats also enough to be done as part of this pr